app/mm-redux/utils/helpers.ts

```diff
@@ -121,3 +121,21 @@ export function buildQueryString(parameters: Dictionary<any>): string {
 
     return query;
 }
+
+export function sanitizeCommandForAutocompleteTracking(userInput: string): string {
+    const commandList = ['agenda', 'autolink', 'away', 'bot-server', 'code', 'collapse',
```

Comment:
@iomodo Just curious - are these the commands that are "whitelisted" for autocompletion?  Do we need to add anything in the future if say, a custom slash command was added?

Reply:
@aaronrothschild The tracking is only for mobile. We need to understand whether we need caching or not right?
These are all the slash commands we have on the community. Bunch of them does not make sense for mobile(for example `shortcuts`), but I still included them just to track if someone is trying to use them on mobile.